Library/Homebrew/os.rb

```diff
@@ -21,6 +21,7 @@ def self.linux?
     end
     PATH_OPEN = "/usr/bin/open".freeze
   elsif OS.linux?
+    require "os/linux"
```

Comment:
Does this need to be required globally? I wish we didn't do that for `os/mac`

Reply:
No, it doesn't. I was just following the same pattern. Fixed.